libs/components/common/src/lib/text/slate-utils.ts

```diff
@@ -958,7 +959,41 @@ class SlateUtils {
     }
 
     public getNodeByPath(path: Path) {
-        Editor.node(this.editor, path);
+        return Editor.node(this.editor, path);
+    }
+
+    public getNodeByRange(range: Range) {
+        return Editor.node(this.editor, range);
+    }
+
+    // This may should write with logic of render slate
+    public convertLeaf2Html(textValue: any) {
+        const { text, fontColor, fontBgColor } = textValue;
```

Comment:
Need to escape HTML special chars to prevent security problems

Reply:
Here is exporting html, escape should be done by the user who is use the clipData